admin-notice/class-admin-notice-controller.php

```diff
@@ -31,9 +31,9 @@ public function display_notices() {
 	}
 
 	public function enqueue_scripts() {
-		wp_enqueue_script( 'vip-admin-notice-script', plugins_url( '/js/script.js', __FILE__ ), [], '1.0' );
+		wp_enqueue_script( 'vip-admin-notice-script', plugins_url( '/js/script.js', __FILE__ ), [], '1.0', true );
```

Comment:
> In footer ($in_footer) is not set explicitly wp_enqueue_script; It is recommended to load scripts in the footer. Please set this value to `true` to load it in the footer, or explicitly `false` if it should be loaded in the header.

I have verified that the script has no side-effects other than registering an `onload` handler; it should be safe to be included in the site footer.